src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -27,8 +27,11 @@
 #if CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
 
 #include <platform/internal/BLEManager.h>
+#define RSI_BLE_ENABLE 1
```

Comment:
This define should be removed. It is already being defined in the `wfx_sl_blie_init.h` header and in the sdk `.gni` files.

Reply:
Yeah fixed